bulbs/recirc/mixins.py

```diff
@@ -41,13 +41,25 @@ def save(self, *args, **kwargs):
     def get_recirc_content(self, published=True, count=3):
         """gets the first 3 content objects in the `included_ids`
         """
+        query = self.get_query()
+
+        # check if query has included_ids & if there are any ids in it
+        try:
+            if len(query['included_ids']) == 0:
+                raise KeyError
```

Comment:
ah, yeah. my thought for both cases is that it could have just been constructed and not have an `included_ids` array in the query, or someone could have removed all of the `included_ids` from it.